sdk/messaging/azeventhubs/internal/amqpwrap/amqp_error.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package amqpwrap
+
+import (
+	"errors"
+)
+
+// Error is a wrapper that has the context of which connection and
+// link the error happened with.
+type Error struct {
```

Comment:
Is this only used internally, or will this be returned from APIs?

Reply:
Internally only (and we're in an internal package).